transmogrifier/sources/transformer.py

```diff
@@ -251,17 +253,20 @@ def _transform(self, source_record: dict[str, JSON] | Tag) -> TimdexRecord | Non
             **self.get_required_fields(source_record),
             **optional_fields,
         }
+        fields = self.create_dates_and_locations_from_publishers(fields)
```

Comment:
It's a minor nit, but I'd propose a blank line above this method call and after the `fields = {...}` call.  That previous call is a MAJOR one, where the bulk of the transform happens.  

This `self.create_dates_and_locations_from_publishers()` call -- as discussed below -- is probably pretty temporary, and secondary, and feels like it needs some space.  

I was going to propose some commenting in this method, but the more I look at it, and the more we kind of push its current architecture, it seems clear that a refactor will rework this fairly heavily at some point.

Reply:
Good call, and agree on the need for a future refactor